submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/query/SubmissionListApiTest.kt

```diff
@@ -228,6 +228,17 @@ class SubmissionListApiTest(
         }
     }
 
+    @Test
+    fun `latest updated submission should appear first`() {
+        webClient.submitSingle(getSimpleSubmission(19), TSV)
```

Comment:
I do not see the point of creating 19 submissions and then asserting only the first one. It will be a much better test if we create 3 and verity they are retrrieved in that order.

Reply:
We're already creating them for all the other tests. The aim of this one is to validate that the latest modified, submission 19, which is already created, is always the first appearance in the list.

It might as well be submission 2 or 24. The result should be the same

Reply:
yes exactly what is the point of testing 19 usbmission?.  I think the idea behind test is to have the **simplest**  posible scenario that test what we require.  Using 19 submission instead of 2, or using 10 files instead of two 2 when they do nothing  actually does not help but rather make test harder to follow and harder to understand. 

why having more complex when simple do the job? same principle for me apply for attributes, sections, user. In general I would rather favor simplicity. Now if we are coding a performance test or a mongo documents limits test story is completly diferent but this is not what we are doing here.

Reply:
I think the reason to have 20 submissions in this test is to validate the default limits and offsets we currently have. We could set a custom limit for the test but, I think the idea of the itests is to keep it as similar as possible to the actual use case.